### DISCUSSION TITLE
Is it possible to receive many RTSP streams and turn them into ONVIF ones to be delivered into an NVR

### DISCUSSION THREAD

**Discussion by martorell-jpg** [bluenviron/mediamtx#3134] [Questions]
### Question

Hello. I have a server that "decrypts" propietary cameras and makes them available as an RTSP stream, the problem is that the NVR I want to use is only compatible with ONVIF aside from other brands (which don't apply). Is it possible to reconvert all the local streams, and offer them into a series of channels using ONVIF?

**Comment by aler9** [comment]
Hello, outputting ONVIF streams is still a work in progress and is tracked here: #1402

[a comment by an automated account is not shown]
